### ISSUE TITLE
Problem with loss_mask!!!

### ISSUE THREAD

**Issue by 294486709** [open-mmlab/mmdetection#6335]
**I'm using the default solo config to train my own instance segmentation dataset. For the first few epoch, I noticed a sizeable decress on loss_cls, while loss_mask stays until it reached epoch No.3.**

2021-10-21 11:49:10,451 - mmdet - INFO - Epoch [1][5/66]	lr: 2.005e-05, eta: 0:08:16, time: 0.630, data_time: 0.428, memory: 1912, loss_mask: 2.9405, loss_cls: 1.1468, loss: 4.0873
2021-10-21 11:49:11,590 - mmdet - INFO - Epoch [1][10/66]	lr: 4.505e-05, eta: 0:05:35, time: 0.227, data_time: 0.008, memory: 2349, loss_mask: 2.9406, loss_cls: 1.0447, loss: 3.9853
2021-10-21 11:49:12,653 - mmdet - INFO - Epoch [1][15/66]	lr: 7.004e-05, eta: 0:04:37, time: 0.213, data_time: 0.008, memory: 2349, loss_mask: 2.9401, loss_cls: 0.8401, loss: 3.7802
2021-10-21 11:49:13,750 - mmdet - INFO - Epoch [1][20/66]	lr: 9.504e-05, eta: 0:04:09, time: 0.220, data_time: 0.008, memory: 2349, loss_mask: 2.9402, loss_cls: 0.6848, loss: 3.6250
2021-10-21 11:49:14,943 - mmdet - INFO - Epoch [1][25/66]	lr: 1.200e-04, eta: 0:03:54, time: 0.238, data_time: 0.008, memory: 2349, loss_mask: 2.9405, loss_cls: 0.6631, loss: 3.6036
2021-10-21 11:49:16,073 - mmdet - INFO - Epoch [1][30/66]	lr: 1.450e-04, eta: 0:03:42, time: 0.226, data_time: 0.009, memory: 2349, loss_mask: 2.9403, loss_cls: 0.5505, loss: 3.4908
2021-10-21 11:49:17,190 - mmdet - INFO - Epoch [1][35/66]	lr: 1.700e-04, eta: 0:03:33, time: 0.225, data_time: 0.009, memory: 2349, loss_mask: 2.9403, loss_cls: 0.3735, loss: 3.3138
2021-10-21 11:49:18,319 - mmdet - INFO - Epoch [1][40/66]	lr: 1.950e-04, eta: 0:03:27, time: 0.226, data_time: 0.008, memory: 2349, loss_mask: 2.9402, loss_cls: 0.4397, loss: 3.3799
2021-10-21 11:49:19,396 - mmdet - INFO - Epoch [1][45/66]	lr: 2.200e-04, eta: 0:03:20, time: 0.215, data_time: 0.008, memory: 2349, loss_mask: 2.9395, loss_cls: 0.3355, loss: 3.2749
2021-10-21 11:49:20,508 - mmdet - INFO - Epoch [1][50/66]	lr: 2.450e-04, eta: 0:03:16, time: 0.222, data_time: 0.008, memory: 2349, loss_mask: 2.9385, loss_cls: 0.2716, loss: 3.2102
2021-10-21 11:49:21,604 - mmdet - INFO - Epoch [1][55/66]	lr: 2.700e-04, eta: 0:03:11, time: 0.219, data_time: 0.008, memory: 2349, loss_mask: 2.9394, loss_cls: 0.2705, loss: 3.2099
2021-10-21 11:49:22,717 - mmdet - INFO - Epoch [1][60/66]	lr: 2.950e-04, eta: 0:03:08, time: 0.223, data_time: 0.008, memory: 2349, loss_mask: 2.9396, loss_cls: 0.3387, loss: 3.2783
2021-10-21 11:49:23,758 - mmdet - INFO - Epoch [1][65/66]	lr: 3.200e-04, eta: 0:03:04, time: 0.208, data_time: 0.008, memory: 2349, loss_mask: 2.9383, loss_cls: 0.2951, loss: 3.2334

2021-10-21 11:49:29,162 - mmdet - INFO - Epoch [2][5/66]	lr: 3.500e-04, eta: 0:03:21, time: 0.676, data_time: 0.429, memory: 2425, loss_mask: 2.9398, loss_cls: 0.6572, loss: 3.5970
2021-10-21 11:49:30,347 - mmdet - INFO - Epoch [2][10/66]	lr: 3.750e-04, eta: 0:03:18, time: 0.238, data_time: 0.009, memory: 2425, loss_mask: 2.9390, loss_cls: 0.4391, loss: 3.3781
2021-10-21 11:49:31,490 - mmdet - INFO - Epoch [2][15/66]	lr: 4.000e-04, eta: 0:03:14, time: 0.229, data_time: 0.009, memory: 2425, loss_mask: 2.9388, loss_cls: 0.8707, loss: 3.8095
2021-10-21 11:49:32,690 - mmdet - INFO - Epoch [2][20/66]	lr: 4.250e-04, eta: 0:03:11, time: 0.241, data_time: 0.009, memory: 2425, loss_mask: 2.9395, loss_cls: 0.6427, loss: 3.5822
2021-10-21 11:49:33,847 - mmdet - INFO - Epoch [2][25/66]	lr: 4.500e-04, eta: 0:03:08, time: 0.231, data_time: 0.008, memory: 2425, loss_mask: 2.9381, loss_cls: 0.5689, loss: 3.5070
2021-10-21 11:49:34,959 - mmdet - INFO - Epoch [2][30/66]	lr: 4.750e-04, eta: 0:03:05, time: 0.223, data_time: 0.008, memory: 2425, loss_mask: 2.9379, loss_cls: 0.3612, loss: 3.2992
2021-10-21 11:49:36,065 - mmdet - INFO - Epoch [2][35/66]	lr: 5.000e-04, eta: 0:03:02, time: 0.221, data_time: 0.008, memory: 2425, loss_mask: 2.9380, loss_cls: 0.3020, loss: 3.2400
2021-10-21 11:49:37,173 - mmdet - INFO - Epoch [2][40/66]	lr: 5.000e-04, eta: 0:03:00, time: 0.222, data_time: 0.009, memory: 2425, loss_mask: 2.9394, loss_cls: 0.4904, loss: 3.4298
2021-10-21 11:49:38,271 - mmdet - INFO - Epoch [2][45/66]	lr: 5.000e-04, eta: 0:02:57, time: 0.220, data_time: 0.008, memory: 2425, loss_mask: 2.9360, loss_cls: 0.9279, loss: 3.8639
2021-10-21 11:49:39,352 - mmdet - INFO - Epoch [2][50/66]	lr: 5.000e-04, eta: 0:02:55, time: 0.216, data_time: 0.008, memory: 2425, loss_mask: 2.9348, loss_cls: 1.3609, loss: 4.2956
2021-10-21 11:49:40,380 - mmdet - INFO - Epoch [2][55/66]	lr: 5.000e-04, eta: 0:02:52, time: 0.206, data_time: 0.007, memory: 2425, loss_mask: 2.9372, loss_cls: 0.8799, loss: 3.8171
2021-10-21 11:49:41,489 - mmdet - INFO - Epoch [2][60/66]	lr: 5.000e-04, eta: 0:02:50, time: 0.222, data_time: 0.008, memory: 2425, loss_mask: 2.9349, loss_cls: 0.6818, loss: 3.6167
2021-10-21 11:49:42,642 - mmdet - INFO - Epoch [2][65/66]	lr: 5.000e-04, eta: 0:02:48, time: 0.230, data_time: 0.008, memory: 2425, loss_mask: 2.9355, loss_cls: 0.5926, loss: 3.5281

2021-10-21 11:49:47,900 - mmdet - INFO - Epoch [3][5/66]	lr: 5.000e-04, eta: 0:02:54, time: 0.629, data_time: 0.421, memory: 2505, loss_mask: 2.9334, loss_cls: 0.4655, loss: 3.3989
2021-10-21 11:49:49,000 - mmdet - INFO - Epoch [3][10/66]	lr: 5.000e-04, eta: 0:02:51, time: 0.220, data_time: 0.008, memory: 2505, loss_mask: 2.9315, loss_cls: 0.3270, loss: 3.2585
2021-10-21 11:49:50,212 - mmdet - INFO - Epoch [3][15/66]	lr: 5.000e-04, eta: 0:02:50, time: 0.241, data_time: 0.008, memory: 2505, loss_mask: 2.9341, loss_cls: 0.2920, loss: 3.2261
2021-10-21 11:49:51,313 - mmdet - INFO - Epoch [3][20/66]	lr: 5.000e-04, eta: 0:02:47, time: 0.221, data_time: 0.009, memory: 2505, loss_mask: 2.9281, loss_cls: 0.2550, loss: 3.1831
2021-10-21 11:49:52,427 - mmdet - INFO - Epoch [3][25/66]	lr: 5.000e-04, eta: 0:02:45, time: 0.223, data_time: 0.008, memory: 2505, loss_mask: 2.9294, loss_cls: 0.2290, loss: 3.1585
2021-10-21 11:49:53,539 - mmdet - INFO - Epoch [3][30/66]	lr: 5.000e-04, eta: 0:02:43, time: 0.222, data_time: 0.008, memory: 2505, loss_mask: 2.9275, loss_cls: 0.2286, loss: 3.1561
2021-10-21 11:49:54,673 - mmdet - INFO - Epoch [3][35/66]	lr: 5.000e-04, eta: 0:02:41, time: 0.226, data_time: 0.008, memory: 2505, loss_mask: 2.9264, loss_cls: 0.2362, loss: 3.1626
2021-10-21 11:49:55,743 - mmdet - INFO - Epoch [3][40/66]	lr: 5.000e-04, eta: 0:02:39, time: 0.214, data_time: 0.008, memory: 2505, loss_mask: 2.9185, loss_cls: 0.2352, loss: 3.1537
2021-10-21 11:49:56,908 - mmdet - INFO - Epoch [3][45/66]	lr: 5.000e-04, eta: 0:02:38, time: 0.232, data_time: 0.009, memory: 2505, loss_mask: 2.9261, loss_cls: 0.2351, loss: 3.1612
2021-10-21 11:49:57,974 - mmdet - INFO - Epoch [3][50/66]	lr: 5.000e-04, eta: 0:02:36, time: 0.214, data_time: 0.009, memory: 2505, loss_mask: 2.9101, loss_cls: 0.1670, loss: 3.0771
2021-10-21 11:49:59,047 - mmdet - INFO - Epoch [3][55/66]	lr: 5.000e-04, eta: 0:02:34, time: 0.215, data_time: 0.008, memory: 2505, loss_mask: 2.9081, loss_cls: 0.2111, loss: 3.1192
2021-10-21 11:50:00,169 - mmdet - INFO - Epoch [3][60/66]	lr: 5.000e-04, eta: 0:02:32, time: 0.224, data_time: 0.008, memory: 2505, loss_mask: 2.8966, loss_cls: 0.2533, loss: 3.1499
2021-10-21 11:50:01,297 - mmdet - INFO - Epoch [3][65/66]	lr: 5.000e-04, eta: 0:02:30, time: 0.226, data_time: 0.008, memory: 2505, loss_mask: 2.8804, loss_cls: 0.2073, loss: 3.0878

Also, when i purposely changed the learning rate to a vary large number. I noticed the loss_cls exploded as expected, but loss_mask almost stays the same. 

2021-10-21 12:00:34,634 - mmdet - INFO - Epoch [1][5/66]	lr: 4.010e-03, eta: 1:13:46, time: 0.671, data_time: 0.428, memory: 2356, loss_mask: 2.9403, loss_cls: 4.8064, loss: 7.7467
2021-10-21 12:00:35,733 - mmdet - INFO - Epoch [1][10/66]	lr: 9.009e-03, eta: 0:49:00, time: 0.221, data_time: 0.010, memory: 2356, loss_mask: 2.9396, loss_cls: 19.2395, loss: 22.1791
2021-10-21 12:00:36,881 - mmdet - INFO - Epoch [1][15/66]	lr: 1.401e-02, eta: 0:41:00, time: 0.229, data_time: 0.007, memory: 2356, loss_mask: 2.9380, loss_cls: 14.0345, loss: 16.9725
2021-10-21 12:00:37,980 - mmdet - INFO - Epoch [1][20/66]	lr: 1.901e-02, eta: 0:36:46, time: 0.221, data_time: 0.009, memory: 2356, loss_mask: 2.9738, loss_cls: nan, loss: nan

Is there something wrong with my setting?
